composer.json

```diff
@@ -17,14 +17,16 @@
     ],
     "require": {
         "php": "~7.0",
-        "guzzle/guzzle": "~3.6",
-        "guzzlehttp/guzzle": "~6.0",
-        "zendframework/zend-diactoros": "~1.1",
-        "container-interop/container-interop": "^1.1"
+        "container-interop/container-interop": "^1.1",
+        "guzzle/guzzle": "^3.9",
+        "guzzlehttp/guzzle": "^6.1",
```

Comment:
We explicity use it in [TokenExchanger](https://github.com/zf-fr/zfr-shopify/blob/master/src/OAuth/TokenExchanger.php#L5)